braph2genesis/graphs/_MultilayerBU.gen.m

```diff
@@ -0,0 +1,433 @@
+    M = binarize(M); % enforces binary adjacency matrix, equivalent to binarize(M, 'threshold', 0, 'bins', [-1:.001:1])
+    B(i, i) = {M};
+end
+A = B;
+value = A;
+
+%%%% ¡gui!
+pr = PanelPropCell('EL', g, 'PROP', MultilayerBU.A, ...
+    'TABLE_HEIGHT', s(40), ...
+    'XYSLIDERLOCK', true, ... 
+    'XSLIDERSHOW', false, ...
+    'YSLIDERSHOW', true, ...
+    'YSLIDERLABELS', g.getCallback('ALAYERLABELS'), ...
+    'YSLIDERWIDTH', s(5), ...
+    'ROWNAME', g.getCallback('ANODELABELS'), ...
+    'COLUMNNAME', g.getCallback('ANODELABELS'), ...
+    varargin{:});
+
+%%% ¡prop!
+PARTITIONS (result, rvector) returns the number of layers in the partitions of the graph.
+%%%% ¡calculate!
+value = ones(1, g.get('LAYERNUMBER'));
+
+%%% ¡prop!
+ALAYERLABELS (query, stringlist) returns the layer labels to be used by the slider.
+%%%% ¡calculate!
+alayerlabels = g.get('LAYERLABELS');
+if isempty(alayerlabels) && ~isa(g.getr('A'), 'NoValue') % ensures that it's not unecessarily calculated
+    alayerlabels = cellfun(@num2str, num2cell([1:1:g.get('LAYERNUMBER')]), 'uniformoutput', false);
+end
+value = alayerlabels;
+
+%%% ¡prop!
+COMPATIBLE_MEASURES (constant, stringlist) is the list of compatible measures.
+%%%% ¡default!
+getCompatibleMeasures('MultilayerBU')
+
+%% ¡props!
+
+%%% ¡prop!
+B (data, cell) is the input cell containing the multilayer adjacency matrices on the diagonal.
+%%%% ¡default!
+{[] []; [] []}
+%%%% ¡gui!
+pr = PanelPropCell('EL', g, 'PROP', MultilayerBU.B, ...
+    'TABLE_HEIGHT', s(40), ...
+    'XSLIDERSHOW', true, ...
+    'XSLIDERLABELS', g.get('LAYERLABELS'), ...
+    'XSLIDERHEIGHT', s(3.5), ...
+    'YSLIDERSHOW', false, ...
+    'ROWNAME', g.getCallback('ANODELABELS'), ...
+    'COLUMNNAME', g.getCallback('ANODELABELS'), ...
+    varargin{:});
+
+%%% ¡prop!
+SYMMETRIZE_RULE (parameter, option) determines how to symmetrize the matrix.
+%%%% ¡settings!
+{'max', 'sum', 'average', 'min'}
+
+%%% ¡prop!
+SEMIPOSITIVIZE_RULE (parameter, option) determines how to remove the negative edges.
+%%%% ¡settings!
+{'zero', 'absolute'}
+
+%% ¡tests!
+
+%%% ¡excluded_props!
+[MultilayerBU.PFGA MultilayerBU.PFGH]
+
+%%% ¡test!
+%%%% ¡name!
+Constructor - Full
+%%%% ¡probability!
+.01
+%%%% ¡code!
+B1 = rand(randi(10));
+B2 = rand(randi(10));
+B3 = rand(randi(10));
+B12 = rand(size(B1,1),size(B2,2));
+B13 = rand(size(B1,1),size(B3,2));
+B23 = rand(size(B2,1),size(B3,2));
+B = {
+    B1                           B12                            B13
+    B12'                         B2                             B23
+    B13'                         B23'                           B3
+    };
+g = MultilayerBU('B', B);
+g.get('A_CHECK')
+A1 = symmetrize(binarize(semipositivize(dediagonalize(B1))));
+A2 = symmetrize(binarize(semipositivize(dediagonalize(B2))));
+A3 = symmetrize(binarize(semipositivize(dediagonalize(B3))));
+B{1,1} = A1;
+B{2,2} = A2;
+B{3,3} = A3;
```

Comment:
leave a space after comma, e.g. 
B12 = rand(size(B1, 1), size(B2, 2));

And spread it across the whole script